src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -75,7 +75,7 @@ public async Task<DiagnosticAnalysisResult> GetAnalysisDataAsync(IPersistentStor
                 }
 
                 // loading data can be cancelled any time.
-                var serializer = new DiagnosticDataSerializer(_owner.AnalyzerVersion, lastResult.Version);
+                var serializerVersion = lastResult.Version;
```

Comment:
in general, where we used to use a Serializer (which points at a 'Version' and assocaited teh diagnostics with that version), we now just pas along the version instead.